index.ts

```diff
@@ -1,32 +1,113 @@
+import { v4 as uuidv4 } from "uuid";
+
 /**
  * This produces an Airplane output (`value`) that is grouped with
  * other outputs with the same `key`. If a `key` is not provided,
  * a default key is used.
- * 
+ *
  * To learn more about Airplane outputs, see the docs: https://docs.airplane.dev/reference/outputs
- * 
+ *
+ * @deprecated please use setOutput and appendOutput instead
+ *
  * @example
  *   output("Task completed successfully.")
- * 
+ *
  * @example
  *   output("error", "This is an error message")
- * 
+ *
  * @example
  *   output("rows", { "name": "Carolyn", "occupation": "agent" })
  */
 function output(keyOrValue: string | any, value?: any) {
   if (arguments.length === 1) {
     const output = keyOrValue === undefined ? null : JSON.stringify(keyOrValue);
-    console.log(`airplane_output ${output}`);
+    logChunks(`airplane_output ${output}`);
   } else {
     if (typeof keyOrValue !== "string") {
       throw new Error("Expected keyOrValue to be type string");
     }
     const output = value === undefined ? null : JSON.stringify(value);
-    console.log(`airplane_output:"${keyOrValue}" ${output}`);
+    logChunks(`airplane_output:"${keyOrValue}" ${output}`);
+  }
+}
+/**
+ * This sets the Airplane output (`value`). If a path is provided, it sets the
+ * portion of the output described by the path. For path syntax, please see the docs.
+ *
+ * To learn more about Airplane outputs, see the docs: https://docs.airplane.dev/reference/outputs
+ *
+ * @example
+ *   setOutput("Task completed successfully.")
+ *
+ * @example
+ *   setOutput({"values": {"error": "This is an error message"}})
```

Comment:
Let's use real world examples. It makes it easier to connect with how you would use in practice :)

```suggestion
 *   setOutput({"error": "Received a 500 from the upstream API"})
```